packages/bpk-component-select/src/BpkSelect-test.js

```diff
@@ -42,6 +45,52 @@ describe('BpkSelect', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render correctly with image', () => {
+    const tree = renderer
+      .create(
+        <BpkSelect
+          id="fruits"
+          name="fruits"
+          value="oranges"
+          image={imageRenderProp}
+          onChange={() => null}
+        >
+          <option value="apples">Apples</option>
+          <option value="oranges">Oranges</option>
+          <option value="pears">Pears</option>
+          <option value="tomatos" disabled>
+            Tomatos
+          </option>
+        </BpkSelect>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly with image and custom classes', () => {
+    const tree = renderer
+      .create(
+        <BpkSelect
+          id="fruits"
+          name="fruits"
+          value="oranges"
+          imageContainerClassName="imageContainerClass--test"
+          wrapperClassName="wrapperClass--test"
+          image={imageRenderProp}
+          onChange={() => null}
+        >
+          <option value="apples">Apples</option>
+          <option value="oranges">Oranges</option>
+          <option value="pears">Pears</option>
+          <option value="tomatos" disabled>
+            Tomatos
```

Comment:
I know this is just a test file, but it's `tomatoes` with an `e` 🙊

Reply:
well then..

Reply:
it was a test wide typo, not my fault 😄